docs/visual-basic/programming-guide/language-features/strings/converting-between-strings-and-other-data-types.md

```diff
@@ -1,25 +1,27 @@
 ---
-title: "Converting Between Strings and Other Data Types in Visual Basic"
+title: "Converting between strings and other data types in Visual Basic"
 ms.date: 07/20/2015
-helpviewer_keywords: 
+helpviewer_keywords:
   - "Visual Basic, strings"
   - "data types [Visual Basic], converting"
   - "conversions [Visual Basic], Visual Basic"
   - "string conversion [Visual Basic], Visual Basic"
 ms.assetid: 06085ed5-191d-4930-8fbc-da63930da400
 ---
-# Converting Between Strings and Other Data Types in Visual Basic
-This section describes how to convert strings into other data types.  
-  
-## In This Section  
- [How to: Convert an Array of Bytes into a String in Visual Basic](../../../../visual-basic/programming-guide/language-features/strings/how-to-convert-an-array-of-bytes-into-a-string.md)  
+# Converting between strings and other data types in Visual Basic
+
+This section describes how to convert strings into other data types.
+
+## In This Section
```

Comment:
this list is actually missing the two last topics from this section. Would you like to add them?

https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/language-features/strings/converting-between-strings-and-other-data-types

![image](https://user-images.githubusercontent.com/12971179/65836043-aefe6e00-e2a2-11e9-9e0a-46e1409270f0.png)

Reply:
@mairaw, Sure. I'll add them.

Reply:
@mairaw, I've added them. I noticed that some file names contains small words like `a`, `to`, .. I believe this shouldn't be like that. Am I correct ?

Reply:
Yeah, but when we migrated content from MSDN, the migration process wasn't aware of these rules. But not sure if it's worth changing now but we do pay attention when creating new articles or moving them.